scripts/set_deployment_image_tag.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I wonder if this should be [with the helm chart](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/master/charts/rstudio/migrations/20180411-1155_increase_cookie_maxage.sh) or not.